src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -1793,8 +1793,8 @@ internal List<CommandMetadata> GetRemoteCommandMetadata(out Dictionary<string, s
         private void WriteProgress(string statusDescription, int? percentComplete, int? secondsRemaining)
         {
             ProgressRecordType recordType;
-            if (secondsRemaining.HasValue && secondsRemaining.Value == 0 &&
-                percentComplete.HasValue && percentComplete.Value == 100)
+            if (secondsRemaining.HasValue && secondsRemaining.GetValueOrDefault() == 0 &&
+                percentComplete.HasValue && percentComplete.GetValueOrDefault() == 100)
```

Comment:
@iSazonov I don't understand this change. `HasValue` check has been done, so it's fine to directly use `.Value`, isn't it?

Reply:
Come from https://github.com/dotnet/runtime/issues/33792